invenio_rdm_records/services/communities/service.py

```diff
@@ -325,3 +325,26 @@ def search_suggested_communities(
             extra_filter=communities_filter,
             **kwargs,
         )
+
+    @unit_of_work()
+    def set_default(self, identity, id_, data, uow):
+        """Set default community."""
+        record = self.record_cls.pid.resolve(id_)
+
+        self.require_permission(identity, "manage", record=record)
+
+        errors = []
+        try:
+            record.parent.communities.default = data["data"]
+        except AttributeError as ex:
+            errors.append({"message": str(ex)})
```

Comment:
Major: Connected to the comment on the resource layer, the errors we do only want to control if we are processing multiple operations, whichc doesn't seem the case here, I would just raise the exception and let the error_handler take care of it. Meaning that there is no need to control the errors here